app/main.py

```diff
@@ -1,6 +1,23 @@
+import requests
+import os
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+API_KEY = os.environ.get("API_KEY")
+params = F"{URL}?key={API_KEY}&q={CITY}&aqi=no"
```

Comment:
https://requests.readthedocs.io/en/latest/user/quickstart/#:~:text=Passing%20Parameters%20In%20URLs
don't do this through the f-string - read the docs

Reply:
ready